Sources/AuthFoundation/OAuth2/Configuration.swift

```diff
@@ -50,50 +51,20 @@ extension OAuth2Client {
         public init(issuerURL: URL,
                     discoveryURL: URL? = nil,
                     clientId: String,
-                    scope: String,
+                    scope: ClaimCollection<[String]>,
                     redirectUri: URL? = nil,
                     logoutRedirectUri: URL? = nil,
                     authentication: ClientAuthentication = .none)
         {
-            var relativeURL = issuerURL
-            
-            // Ensure the base URL contains a trailing slash in its path, so request paths can be safely appended.
-            if !relativeURL.lastPathComponent.isEmpty {
-                relativeURL = relativeURL.appendingComponent("")
-            }
-            
             self.issuerURL = issuerURL
-            self.discoveryURL = discoveryURL ?? relativeURL.appendingComponent(".well-known/openid-configuration")
+            self.discoveryURL = discoveryURL ?? issuerURL.appendingDiscoveryURL
```

Comment:
Any concern of trailing slashes here? It seems logic was removed

Reply:
It was moved to Sources/AuthFoundation/Utilities/URL+InternalExtensions.swift so this logic could be cleaner within the initializer.